erpnext_documentation/www/docs/user/manual/en/manufacturing/subcontracting.md

```diff
@@ -61,6 +61,15 @@ Click on Create > Purchase Receipt from the Purchase Order. Set the Accepted and
 
 <img class="screenshot" alt="Sub-Contracting" src="{{docs_base_url}}/assets/img/manufacturing/subcontract5.png">
 
+### 1.5 Supplier Sourced Raw Material
+If some raw material is sourced by the Supplier directly, such raw material has to be marked in the BOM.
```

Comment:
```suggestion
While creating a BOM for subcontracting, there might be raw material which Supplier will have to procure themselves like nuts and bolts. While creating a Stock Entry for Transfer from Purchase Order, these items can be excluded one by one but it is impossible to do so if you have more than 100 items. 

If some raw material is sourced by the Supplier directly, such raw material has to be marked in the BOM.
```

Reply:
Little too descriptive I feel. We do not need this line "While creating a Stock Entry for Transfer from Purchase Order, these items can be excluded one by one but it is impossible to do so if you have more than 100 items. "

Reply:
better to be descriptive than leave doubts :)